README.md

```diff
@@ -65,6 +65,8 @@ If your GPU is not listed here or you do not meet the minimum CC, you will not b
 Aphrodite will require a slightly specialized environment to run, as the latest CUDA and GCC versions are not supported. You can use Conda to easily configure your environment.
 
 ### Install miniconda3
+If you run into any problems, please refer the common [Common Issues](#common-issues) section, or open an [Issue](https://github.com/PygmalionAI/aphrodite-engine/issues) if you can't find the answer there.
```

Comment:
Hate to sound like a grammar nazi but
"Please refer *to* the Common Issues section"